app/_locales/en/messages.json

```diff
@@ -4302,7 +4302,7 @@
     "message": "Priority fee (GWEI)"
   },
   "notificationItemCheckBlockExplorer": {
-    "message": "Check on the Block Explorer"
+    "message": "Check on the block explorer"
```

Comment:
Not sure about this one. Is "Block Explorer" considered a special term @coreyjanssen?